resolwe_bio/processes/alignment/bwa_index.py

```diff
@@ -84,7 +81,7 @@ def run(self, inputs, outputs):
 
         if inputs.alt_file:
             alt_fname = f"{name}.fasta.alt"
-            src = Path(inputs.alt_file.output.file.path)
+            src = Path(inputs.alt_file.import_file(imported_format="extracted"))
             alt_path = Path(alt_fname)
             if src.suffix == ".gz":
```

Comment:
I don't think this part is necessary, as the import function returns the extracted alt file.